src/lib.rs

```diff
@@ -158,6 +158,9 @@ pub struct Settings {
     /// The maximum length of outgoing frames. Messages longer than this will be fragmented.
     /// Default: 65,535
     pub fragment_size: usize,
+    /// The maximum length of acceptable incoming frames. Messages longer than this will be rejected.
+    /// Default: unlimited
+    pub max_fragment_size: usize,
```

Comment:
In the implementation you are using `u64` and here `usize`? Why not use `u64` everywhere?

Reply:
Okay :)